examples/gdp/stickies.py

```diff
@@ -486,7 +488,7 @@ def log9_rule(model, s, sprime):
     # model.accept_or_reject4 = Constraint(model.ScreenNodePairs,
     #                                      rule=accept_or_reject_rule4)
 
-    instance = model.create_instance(DATFILE)
+    instance = model.create_instance(os.path.join(os.path.dirname(__file__), DATFILE))
```

Comment:
Please use `this_file_dir()` from `pyomo.common.fileutils` (it is more reliable than `__file__`):
```python
     instance = model.create_instance(os.path.join(this_file_dir(), DATFILE))
```